src/services/anchor-service.ts

```diff
@@ -101,6 +101,10 @@ export default class AnchorService implements Contextual {
     this.requestService = context.lookup('RequestService');
   }
 
+  public async getChainId(): Promise<string> {
```

Comment:
Why is this method in the `AnchorService`? You can get the value in the controller from the `BlockchainService`, right?

Reply:
I guess I would have thought that would be a layering violation?  I had assumed that the BlockchainService would be an implementation detail of the AnchorService.  Currently I don't believe there are any places that access the BlockchainService outside of the AnchorService, so this would be the first.

Anyway I don't feel super strongly about this, if you prefer to get rid of this method and use the blockchain service from `ServiceInfoController` directly I can do that.

Reply:
New version accesses the blockchain service directly